.github/workflows/build-verification.yml

```diff
@@ -25,3 +25,69 @@ jobs:
         run: ./gradlew build -i
         env:
           GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GE_SOLUTIONS_ACCESS_TOKEN }}
+      - name: Extract build validation Scripts
```

Comment:
We should add the same verification steps to the `development-release.yml` workflow which runs on master.

Reply:
I will address CI in a separate PR. I have removed these changes for now.